spec/controllers/book_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+require 'support/shared_context'
+
+describe BookController, type: :controller do
```

Comment:
Thanks for doing some test. It's a very good practice to test everything, but here this wasn't asked ;)

Reply:
Also, i will not correct this in details, I let you some comments, but we will se the rest when the card come.

Reply:
I made a mistake and did most of the testing card on this branch instead of a new one without noticing.